pages/about.js

```diff
@@ -1,7 +1,10 @@
+// import Header from "../comps/Header";
+import Layout from "../components/MyLayout";
+
 export default function About() {
   return (
-    <div>
+    <Layout>
       <p>This is the about page</p>
-    </div>
+    </Layout>
   );
 }
```

Comment:
Based on the above this could be:

```js
const About = () => (
  <Layout>
    <p>This is the about page</p>
  </Layout>
);

export default About;
```

Two things to note in this case:

1. If your fat arrow function takes no arguments, you **must** include the parenthesis `() => {}`
2. You can't `export default const foo = 'bar';` because the runtime can't guarantee what you're doing in the assignment won't have side effects … it's kind of hard to explain but it has to do with JS being dynamic and the module system relying on things being static (a function declaration is static, but it's invocation might not be … a const is static, but the assignment might not be). All you really need to know is `export default function Blah() {}` is fine, but `export default const Blah = () => {};` is not … got do `const Blah = () => {}; export default Blah;`